.gen/proto/api/v1/common.pb.go

```diff
@@ -1732,10 +1732,7 @@ func (m *WorkflowExecution) Unmarshal(dAtA []byte) error {
 			if err != nil {
 				return err
 			}
-			if skippy < 0 {
-				return ErrInvalidLengthCommon
-			}
-			if (iNdEx + skippy) < 0 {
+			if (skippy < 0) || (iNdEx+skippy) < 0 {
```

Comment:
It seems like this is due to some version upgrade in the proto tool, so I believe that's fine. 
Cc @Groxx

Reply:
This shouldn't be affected unless the makefile's versioned proto tool was changed, which this diff doesn't do.

The two branches that are merged are identical though, possibly a newer `go fmt -s` did this?